backend/src/main/resources/graphql/main.graphqls

```diff
@@ -198,74 +168,48 @@ type TestResult {
   patientLink: PatientLink
 }
 
-# Note: we assume a user's email and login username are the same thing.
-type User {
-  id: ID
-  firstName: String
-  middleName: String
-  lastName: String!
-  suffix: String
-  email: String!
-  isAdmin: Boolean
-  roleDescription: String!
-  permissions: [UserPermission!]!
-  organization: Organization
-  # for backward compatibility, currently maintaining both `role` and `roles`
-  role: Role
-  roles: [Role!]!
-}
-
-type ApiUser {
-  nameInfo: NameInfo
-}
-
-type NameInfo {
-  firstName: String
-  middleName: String
-  lastName: String
-}
-
 type PatientLink {
   internalId: ID
   createdAt: DateTime
   expiresAt: DateTime
   testOrder: TestOrder
 }
 
+# Queries and mutations for everyday use
+
 type Query {
+  organization: Organization
   deviceType: [DeviceType]
   patients(
     facilityId: ID
     pageNumber: Int
     pageSize: Int
-    showDeleted: Boolean = false @requiredPermissions(allOf: ["READ_ARCHIVED_PATIENT_LIST"])
+    showDeleted: Boolean = false
```

Comment:
There's a bunch of VSCode/prettier rearrangement here. My normal rule is to not do those rearrangements, but this is actually a rearrange-and-clean-up story, so I'm less opposed to it than usual. OTOH, if people dislike this rearrangement, I should back it out and fix my VSCode configuration. 😁

Reply:
Tagging @jeskew-gov for directive-managing opinions.

Reply:
Moving directives to a new line looks just as readable to me. It's best to do this now while the file is being split to minimize noise in the git log for any given line.